meinberlin/assets/scss/components/details.scss

```diff
@@ -0,0 +1,18 @@
+details summary {
```

Comment:
@phillimorland Should be _details.scss, right?

Reply:
fuck yes